sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/FunMatcher.kt

```diff
@@ -207,6 +214,7 @@ fun FunMatcher(
     dynamic: Boolean? = null,
     extensionFunction: Boolean? = null,
     suspending: Boolean? = null,
+    operator: Boolean? = null,
```

Comment:
`operator` is a valid variable name but it could be confuse by some developers with the kotlin function modifier keyword. I would rename it with `overloadedOperator`.

Reply:
I would have called it `isOperator`, but this would break the naming schema of the existing arguments (i.e., dynamic instad isDynamic et c.) overloadedOperator, operatorFunction or something like that sounds good.

Reply:
I think that is fine to use `operator`. Also, you can use `isOperator` and refactor all other boolean properties. 
`overloadedOperator` or `operatorFunction` sound more confusing to me.

Reply:
I opted for the refactoring. Fixed.